src/it/it-abstract-versions-report-001/verify.groovy

```diff
@@ -1,15 +1,17 @@
-dependencyUpdatesReport = new File( basedir, "target/site/dependency-updates-report.html" )
-
-assert dependencyUpdatesReport.exists(  )
+dependencyUpdatesReport = new File( basedir, "target/site/dependency-updates-report.html" ).text
+        .replaceAll( '<[^>]+>', ' ' )
+        .replaceAll( '&[^;]+;', ' ' )
+        .replaceAll( '\\s+', ' ' )
 
 // some basic (=not comprehensive) checks on the contents of the dependency report
-assert dependencyUpdatesReport.text =~ /\b1\.1\.0-2\b/  // current version of the dependency
-assert dependencyUpdatesReport.text =~ /\b3\.0\b/       // latest major available version
```

Comment:
I start thinking that we can use external library for parsing html like https://jsoup.org/ ...
But such improvement is for another PR 😄

Reply:
Yes and no. This is faster and more concise, but on the other hand, it is still using regexp which is SLOW. However, I only did it so that, if the test is wrong, it's much easier to see the differences. :)